TiKV config: Update some values and add new online configurations (#7963)
pingcap/docs: tikv-configuration-file.md

### `scheduler-worker-pool-size`

+ The number of threads in the Scheduler thread pool. Scheduler threads are mainly used for checking transaction consistency before data writing. If the number of CPU cores is greater than or equal to `16`, the default value is `8`; otherwise, the default value is `4`. When you modify the size of the Scheduler thread pool, refer to [Performance tuning for TiKV thread pools](/tune-tikv-thread-performance.md#performance-tuning-for-tikv-thread-pools).
+ Default value: `4`

> + Minimum value: `1`

Author: ```suggestion
+ Value range: `[1, MAX(4, CPU)]`. In `MAX(4, CPU)`, `CPU` means the number of your CPU cores. `MAX(4, CPU)` takes the greater value out of `4` and the `CPU`.
+ Default value: MAX(4, CPU * 0.8
```

Author: This part is revised according to [#8784](https://github.com/pingcap/docs-cn/pull/8784)

> ### `scheduler-worker-pool-size`

Author: This part was updated in [#8644](https://github.com/pingcap/docs-cn/pull/8644) and revised in [#8784](https://github.com/pingcap/docs-cn/pull/8784).

+ The number of threads in the Scheduler thread pool. Scheduler threads are mainly used for checking transaction consistency before data writing. If the number of CPU cores is greater than or equal to `16`, the default value is `8`; otherwise, the default value is `4`. When you modify the size of the Scheduler thread pool, refer to [Performance tuning for TiKV thread pools](/tune-tikv-thread-performance.md#performance-tuning-for-tikv-thread-pools).
+ Default value: `4`
+ Value range: `[1, MAX(4, CPU)]`. In `MAX(4, CPU)`, `CPU` means the number of your CPU cores. `MAX(4, CPU)` takes the greater value out of `4` and the `CPU`.

> + Default value: MAX(4, CPU * 0.8)

Author: ```suggestion
```

+ The soft limit on the size of a single message packet
+ Default value: `"1MB"`

> + Minimum value: greater than `0`

Reviewer: ```suggestion
+ Minimum value: Greater than `0`
```

Author: > 

How about using the lower cases as other lines (for example 501, 538)?

+ The soft limit on the size of a single message packet

> + Default value: `"1MB"`

Author: This part was updated in [#8644](https://github.com/pingcap/docs-cn/pull/8644).

+ Minimum value: greater than `0`
+ Maximum value: `3GB`
+ Unit: KB|MB|GB

### `raft-max-inflight-msgs`

+ The number of Raft logs to be confirmed. If this number is exceeded, the Raft state machine slows down log sending.
+ Default value: `256`
+ Minimum value: greater than `0`

> + Maximum value: `16384`

Author: This part was updated in [#8784](https://github.com/pingcap/docs-cn/pull/8784)

### `apply-max-batch-size`

> + Raft state machines execute data write requests in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can execute the requests in one batch.

Reviewer: ```suggestion
+ Raft state machines process data write requests in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can execute the requests in one batch.
```

+ Default value: `256`
+ Minimum value: greater than `0`
+ Maximum value: `10240`

### `apply-pool-size`

> + The allowable number of threads in the pool that flushes data to storage, which is the size of the Apply thread pool. When you modify the size of this thread pool, refer to [Performance tuning for TiKV thread pools](/tune-tikv-thread-performance.md#performance-tuning-for-tikv-thread-pools).

Author: 该修改与 #7964 中的修改重复（修改内容相同）

+ Default value: `2`
+ Minimum value: greater than `0`

### `store-max-batch-size`

> + Raft state machines execute requests for log flushing in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can execute the requests in one batch.

Reviewer: ```suggestion
+ Raft state machines process requests for spilling logs into disk in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can process the requests in one batch.
```

### `apply-max-batch-size`

> + Raft state machines process data write requests in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can execute the requests in one batch.

Reviewer: ```suggestion
+ Raft state machines process data write requests in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can process the requests in one batch.
```

+ Default value: `256`
+ Minimum value: greater than `0`
+ Maximum value: `10240`

### `apply-pool-size`

+ The allowable number of threads in the pool that flushes data to the disk, which is the size of the Apply thread pool. When you modify the size of this thread pool, refer to [Performance tuning for TiKV thread pools](/tune-tikv-thread-performance.md#performance-tuning-for-tikv-thread-pools).
+ Default value: `2`

> + Value ranges: `[1, CPU * 10]` (`CPU` means the number of your CPU cores)

Reviewer: ```suggestion
+ Value ranges: `[1, CPU * 10]`. `CPU` means the number of your CPU cores.
```

### `store-max-batch-size`

+ Raft state machines execute requests for log flushing in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can execute the requests in one batch.
+ If `hibernate-regions` is enabled, the default value is `256`. If `hibernate-regions` is disabled, the default value is `1024`.
+ Minimum value: greater than `0`
+ Maximum value: `10240`

### `store-pool-size`

+ The allowable number of threads in the pool that processes Raft, which is the size of the Raftstore thread pool. When you modify the size of this thread pool, refer to [Performance tuning for TiKV thread pools](/tune-tikv-thread-performance.md#performance-tuning-for-tikv-thread-pools).
+ Default value: `2`

> + Value ranges: [1, CPU * 10] (`CPU` means the number of your CPU cores)

Reviewer: ```suggestion
+ Value ranges: `[1, CPU * 10]`. `CPU` means the number of your CPU cores.
```

> ### `apply-max-batch-size`

Author: This part was updated in [#8644](https://github.com/pingcap/docs-cn/pull/8644).

+ Raft state machines process data write requests in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can execute the requests in one batch.
+ Default value: `256`
+ Minimum value: greater than `0`
+ Maximum value: `10240`

> ### `apply-pool-size`

Author: This part was updated in [#8784](https://github.com/pingcap/docs-cn/pull/8784)

> + The allowable number of threads in the pool that flushes data to storage, which is the size of the Apply thread pool. When you modify the size of this thread pool, refer to [Performance tuning for TiKV thread pools](/tune-tikv-thread-performance.md#performance-tuning-for-tikv-thread-pools).

Author: ```suggestion
+ The allowable number of threads in the pool that flushes data to the disk, which is the size of the Apply thread pool. When you modify the size of this thread pool, refer to [Performance tuning for TiKV thread pools](/tune-tikv-thread-performance.md#performance-tuning-for-tikv-thread-pools).
```

+ Default value: `2`
+ Value ranges: `[1, CPU * 10]` (`CPU` means the number of your CPU cores)

> ### `store-max-batch-size`

Author: This part was updated in [#8644](https://github.com/pingcap/docs-cn/pull/8644) and revised in [#8784](https://github.com/pingcap/docs-cn/pull/8784)

+ Raft state machines execute requests for log flushing in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can execute the requests in one batch.
+ If `hibernate-regions` is enabled, the default value is `256`. If `hibernate-regions` is disabled, the default value is `1024`.
+ Minimum value: greater than `0`
+ Maximum value: `10240`

> ### `store-pool-size`

Author: This part was updated in [#8784](https://github.com/pingcap/docs-cn/pull/8784)
